app/views/lessons/v2_project_submissions/_form.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= turbo_frame_tag 'project_submission_form', class: 'w-full' do %>
-  <%= form_with url: lesson_v2_project_submissions_path(lesson.id), model: project_submission, builder: TailwindFormBuilder do |form| %>
+  <%= form_with url:, model: project_submission, builder: TailwindFormBuilder do |form| %>
```

Comment:
heh, it's been tripping me up too 😆. Final version of this will get rid of that url and just use the model object. I only need the url atm because of the v2 namespace.